src/App.tsx

```diff
@@ -14,8 +14,7 @@ import Home from './Home';
 import ProfileSettings from './ProfileSettings';
 import SignIn from './SignIn';
 
-import CreateTransaction from './transaction/CreateTransaction';
-import ReplaceTransaction from './transaction/ReplaceTransaction';
+import Transaction from './transaction/Transaction';
```

Comment:
Consider calling this CreateTransaction so as not to confuse us with the ethers Transaction object